client/todo-list.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+
+const listEntryStyle = {
```

Comment:
Take a look at Bootstrap's [Card](http://getbootstrap.com/docs/4.0/components/card/) component instead of deviating from the overall style.